src/frontend/core/executor/impl/TriangleCountExecutor.cpp

```diff
@@ -456,6 +545,7 @@ long TriangleCountExecutor::getTriangleCount(int graphId, std::string host, int
                 fileCombinationMutex.unlock();
 
                 if (isAggregateValid) {
+                    triangleCount_logger.log("###COMPOSITE### Found out Aggregate Valid Scenario ", "info");
```

Comment:
This log message is not meaningful. What do you mean by `Aggregate Valid Scenario`?

Reply:
Changed the message

Reply:
Ok